tests/toolkit/position_tests.cc

```diff
@@ -1,7 +1,7 @@
 #include <gtest/gtest.h>
-
 #include <iostream>
 #include <string>
+
```

Comment:
Why add this blank line here in this place?
```suggestion
```

Reply:
I think we tend to separate standard includes from project includes (style guide?). But there is an opposite change above...